pvlib/pvsystem.py

```diff
@@ -827,15 +838,16 @@ def pvwatts_losses(self):
     def pvwatts_ac(self, pdc):
         """
         Calculates AC power according to the PVWatts model using
-        :py:func:`pvwatts_ac`, `self.module_parameters['pdc0']`, and
-        `eta_inv_nom=self.inverter_parameters['eta_inv_nom']`.
+        :py:func:`pvlib.inverter.pvwatts`, `self.module_parameters['pdc0']`,
+        and `eta_inv_nom=self.inverter_parameters['eta_inv_nom']`.
```

Comment:
While we're at it, probably change the single ticks around the module parameters items to double ticks so they render using code formatting.